### PR TITLE
MDS-2464 Convert old NOW screen to new NOW component

### PR DESCRIPTION
**Pull request by nayr974**
This PR removes the old NOW front-end, then extends and consumes the new notice of work component to show a view only version of the form.

Additionally:
The MMS NOW Number is now consumed
There is a button to open the Original Application PDF on a NOW

### CODE REVIEW

File: services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js [hunk 1]
```diff
@@ -137,12 +137,12 @@ export class NoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "tracking_number",
+      sortField: "now_number",
       render: (text, record) => (
-        <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
+        <Link to={router.VIEW_NOTICE_OF_WORK_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
```
Comment (mmumby) [inline services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js]: Is there a view button and a link on the now_number that route to the same place? if so, should we remove this link and just have the view button?

File: services/core-web/src/components/mine/NoticeOfWork/MineNoticeOfWorkTable.js [hunk 2]
```diff
@@ -111,12 +111,12 @@ export class MineNoticeOfWorkTable extends Component {
     {
       title: "NoW No.",
       dataIndex: "nowNum",
-      sortField: "tracking_number",
+      sortField: "now_number",
       render: (text, record) => (
-        <Link to={router.NOTICE_OF_WORK_INITIAL_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
+        <Link to={router.VIEW_NOTICE_OF_WORK_APPLICATION.dynamicRoute(record.key)}>{text}</Link>
       ),
```
Comment (mmumby) [inline services/core-web/src/components/mine/NoticeOfWork/MineNoticeOfWorkTable.js]: same question here.

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js [hunk 3]
```diff
@@ -184,12 +184,12 @@ export class NoticeOfWorkApplication extends Component {
         <div className={this.state.fixedTop ? "steps--header fixed-scroll" : "steps--header"}>
           <div className="inline-flex between">
             <div>
-              <h1>NoW Number: {Strings.EMPTY_FIELD}</h1>
+              <h1>NoW Number: {this.props.noticeOfWork.now_number || Strings.EMPTY_FIELD}</h1>
               {/* update to use application_guid for link once guid is persisted */}
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js]: I think this entire comment block/following <Link> can be removed now, it used to link to the original NOW.

**Comment by mmumby** [comment]
Good work :)

File: services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js (hunk 1 above)
Comment (nayr974) [reply services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js]: It crossed my mind but I see no harm in leaving it either. I like having the links in general.

File: services/core-web/src/components/mine/NoticeOfWork/MineNoticeOfWorkTable.js (hunk 2 above)
Comment (nayr974) [reply services/core-web/src/components/mine/NoticeOfWork/MineNoticeOfWorkTable.js]: It crossed my mind but I see no harm in leaving it either. I like having the links in general.

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js (hunk 3 above)
Comment (nayr974) [reply services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js]: Removed :)

**Pull request merged**